INSTALL.rst

```diff
@@ -6,7 +6,7 @@ Installation Requirements
 
 To install Twisted, you need:
 
-- Python 2.7 (full functionality) or 3.5/3.6/3.7 (subset of functionality).
```

Comment:
I would just simplify this and put: Python 3.5 or higher
If we come across a Python version where Twisted doesn't work, we can update this later.

Reply:
Currently a whole bunch of tests fail on 3.8, so as per our policy we don't say we support it quite yet.